### PR TITLE
Fix #118: remove code block delimiters in generated HTML files

### LINKED ISSUE #118: Extra catala markers in the website examples
The website still features "```catala" in the rendered HTML... I think it should be trimmed. Maybe a side effect of the recent switch to markdown syntax?

### PR DESCRIPTION
**Pull request by EmileRolley**
> Related issue: #118

## Question

~~1. I don't understand why this regexp is needed: https://github.com/CatalaLang/catala/compare/master...EmileRolley:fix-remove-catala-markers?expand=1#diff-173106f4bf5075fa7e6f2d0614d92e15637db5c7a94263d75e98ab553842e8f0R54~~
(_removed_)

## Changelog

* add substitutions in order to remove trailling code block delimiters.
* fix wrong file name in `examples/tutoriel_fr/Makefile`.

### CODE REVIEW

**Comment by denismerigoux** [comment]
> I don't understand why this regexp is needed: https://github.com/CatalaLang/catala/compare/master...EmileRolley:fix-remove-catala-markers?expand=1#diff-173106f4bf5075fa7e6f2d0614d92e15637db5c7a94263d75e98ab553842e8f0R54

Me neither, I think you can remove it.

**Comment by denismerigoux** [comment]
I'm glad the regexes turned out to be less complicated than what I expected ! Can you post a gist of the output to check if it displays correctly in a website setting ?

**Comment by EmileRolley** [comment]
> I'm glad the regexes turned out to be less complicated than what I expected ! Can you post a gist of the output to check if it displays correctly in a website setting ?

Sure: https://gist.github.com/EmileRolley/fc84c4857ee8db0248834daf1a05ab90

**Comment by denismerigoux** [comment]
> Sure: https://gist.github.com/EmileRolley/fc84c4857ee8db0248834daf1a05ab90

Oh I see what you did, smart :)

File: src/catala/literate/html.ml [hunk 1]
```diff
@@ -37,6 +37,34 @@ let raise_failed_pygments (command : string) (error_code : int) : 'a =
...
+  R.substitute ~rex:(R.regexp "<pre>.*\n") ~subst:(function _ -> "<pre>\n")
+
+(** Partial application allowing to remove last code lines of [<td class="code">] and
+    [<td class="linenos">] generated HTML. Basically, remove all code block last lines. *)
+let remove_cb_last_lines : string -> string =
+  R.substitute ~rex:(R.regexp "<a.*\n*</pre>") ~subst:(function _ -> "</pre>")
+
+(** Partial application allowing to substitute operators by their unicode representation. *)
+let substitute_arithmetics_op : string -> string =
+  (* (EmileRolley) NOTE: [date] should match the new ISO format. + I don't get why this must be in
```
Comment (denismerigoux) [inline src/catala/literate/html.ml]: You can delete this date regex, I don't know why it's here. Probably a leftover of something I used to do.

**Review by denismerigoux — APPROVED** [review]

**Pull request merged**